libvirt.go

```diff
@@ -1403,20 +1468,50 @@ func (l *Libvirt) GetBlockIOTune(dom string, disk string) ([]BlockLimit, error)
 		return nil, decodeError(r.Payload)
 	}
 
-	dec := xdr.NewDecoder(bytes.NewReader(r.Payload))
-	result := struct {
-		Limits     []TypedParam
-		ParamCount uint32
-	}{}
-	_, err = dec.Decode(&result)
+	fmt.Println(r.Payload)
```

Comment:
oops?

Reply:
Yep, fixed. Also added a comment to the bytes that add a test for strings.

Reply:
I think you still missed this one